modules/client/src/tokenVoting/interfaces.ts

```diff
@@ -261,3 +274,53 @@ export enum UnwrapTokensStep {
 export type UnwrapTokensStepValue =
   | { key: UnwrapTokensStep.UNWRAPPING; txHash: string }
   | { key: UnwrapTokensStep.DONE };
+
+export type DelegateTokensParams = {
```

Comment:
Won't this be colliding with the interfaces PR?

Reply:
yes, but that's a problem for future Jose, i'll adapt the other PR when this two are merged